frontend/scripts/react-components/static-content/static-content-links.js

```diff
@@ -0,0 +1,42 @@
+
+export default {
```

Comment:
I'm wondering if moving this into router.js makes sense. Seems like the first place us-in-6-months will look for when we need to add more pages. What do you think?

Reply:
I actually struggled deciding where to put this. If we put it in the router, we should split that file.

Reply:
Actually I decided against this, the links have more sense in the `static-content` context. If you take them up to the router it will be confusing. Besides, it's currently a relative import so following the breadcrumbs won't be hard. 🍪

Reply:
In the end when the new nav, refactoring this made more sense. I made this change on the new nav PR